src/cli.ts

```diff
@@ -128,16 +128,30 @@ export async function main(): Promise<void> {
   await runScript(script, scriptPath, tempPath)
 }
 
-const rmrf = (p: string) => p && fs.rmSync(p, { force: true, recursive: true })
+// Safer remove: unlink symlinks, recurse only for real dirs/files
+const safeRemove = (p: string) => {
```

Comment:
1. Seems good, but let's shorten a bit:

const rmrf = (p: string) => {
  if (!p) return
  try {
    fs.lstatSync(p).isSymbolicLink()
      ? fs.unlinkSync(p)
      : fs.rmSync(p, { force: true, recursive: true })
  } catch {}
}

2. Apply `npm build` to update bundles too. We keep them in repo.